app/models/like.rb

```diff
@@ -1,3 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :post_id
+  belongs_to :users
+  belongs_to :posts
+
+  def update_likes_counter
+    Post.update(likes_counter: Post.first.likes.count)
+  end
```

Comment:
- [ ] Post written by particular user should be updated, you can also use increment method here: 

```ruby
def update_post_counter
  author.increment!(:posts_counter)
end
```

- [ ] Same for comment and likes 😉

Reply:
Thank you @ShoiraTa, for helping me.